UH-StudentServices/student_guide
# Feature/doo 38 alter feedback email

Testing:

- Send feedback from various pages
- Send feedback as an anonymous user
- Send feedback as an authenticated user
- Fill various fields and checkboxes
- Verify email subject, body and headers (MailHog), verify that the `Reply-to` header is logical (should equal the email of the user, or `noreply@helsinki.fi` if missing.
